conf/build.manifest

```diff
@@ -177,6 +179,9 @@ sources:
 - name: openseachest
   branch: truenas/master
   repo: https://github.com/truenas/openseachest
+- name: inadyn
+  branch: master
+  repo: https://github.com/troglobit/inadyn.git
```

Comment:
Can we fork it? We need it for tagging as part of the release engineering process

Reply:
Definitely, doing it asap